__fixtures__/playlist.fixture.js

```diff
@@ -1,5 +1,5 @@
 function generatePlaylist (playlistItems = []) {
-  return { items: playlistItems }
```

Comment:
Sure! In this specific case I changed it because the playlist is no longer returned as an object with item property (maybe we can go back to the way it was, if you think it is important), but I changed it here so that the unit tests would pass with the new way the playlist is returned.